commands/object-encoding.md

```diff
@@ -9,9 +9,34 @@ Redis objects can be encoded in different ways:
     - `embstr`, an embedded string, which is an object where the internal simple dynamic string, `sds`, is an unmodifiable string allocated in the same chuck as the object itself.
       `embstr` can be strings with lengths up to the hardcoded limit of `OBJ_ENCODING_EMBSTR_SIZE_LIMIT` or 44 bytes. 
 
-* Lists can be encoded as `ziplist` or `linkedlist`. The `ziplist` is the special representation that is used to save space for small lists.
-* Sets can be encoded as `intset` or `hashtable`. The `intset` is a special encoding used for small sets composed solely of integers.
-* Hashes can be encoded as `ziplist` or `hashtable`. The `ziplist` is a special encoding used for small hashes.
-* Sorted Sets can be encoded as `ziplist` or `skiplist` format. As for the List type small sorted sets can be specially encoded using `ziplist`, while the `skiplist` encoding is the one that works with sorted sets of any size.
+* Lists can be encoded as:
+ 
+    - `linkedlist`, normal list encoding. No longer used, it is an old list encoding.
+    - `ziplist`, Redis <= 6.2, a special encoding used for small lists.
+    - `listpack`, Redis >= 7.0, a special encoding used for small lists.
```

Comment:
We don't really describe the purpose of these. The old text mentioned "that is used to save space". Maybe we should change "special" to "compact"  or "space-efficient"?

```suggestion
    - `ziplist`, Redis <= 6.2, a space-efficient encoding used for small lists.
    - `listpack`, Redis >= 7.0, a space-efficient encoding used for small lists.
```

The same applies to all key types where listpack and ziplist are used.